docs/services/newport_picomotor.rst

```diff
@@ -1,14 +1,45 @@
 Newport Picomotor
 =================
 
+This service controls a Newport Picomotor Motion Controller (see link below for an example of one that has been used with catkit2).
+
+https://www.newport.com/f/open-loop-picomotor-motion-controller
+
+The bottom of the webpage linked above has resources for the user including software downloads, technical notes, and user manuals.
```

Comment:
Please state more clearly whether for example a driver needs to be installed for this device or not.